templates/definition/meter/sunspec-inverter-control.yaml

```diff
@@ -92,7 +82,7 @@ render: |
         source: sequence
         set:
         - source: const
-          value: 1 # off
+          value: 1 # on
           set:
             source: sunspec
             {{- include "modbus" . | indent 10 }}
```

Comment:
Warum wird der hier noch gesetzt? Und wenn er gesetzt werden muss- warum bei den anderen Modi nicht resettet?

Reply:
In der Diskussion hatte es gehießen, dass das Voraussetzung für das Netzladen wäre, für alles andere nicht.

Wenn jemand kein Netzladen verwendet bleibt die lokale Konfiguration dann dahingehend unbeeinflusst.